kind-setup.sh

```diff
@@ -33,7 +33,7 @@ function get_status() {
     }
     { print $f["STATUS"] }
     ' | grep Provisioned > /dev/null
-
+    kubectl get $OBJECT
```

Comment:
Please remove this. This is circumventing the test.

Reply:
Removed